### ISSUE TITLE
Add support for conditional tags

### ISSUE THREAD

**Issue by sheabunge** [sheabunge/code-snippets#135]
One feature we have requested pretty often is the ability to choose exactly when a snippet is evaluated. This could encompass both *where* (i.e., post type, page type, etc) and *when* (logged in, administrator, specific users, etc).

While this can be easily achieved by including conditional tags in Functions snippets, it seems that people would find an interface control helpful, and it would be especially useful for snippets that don't allow for PHP code (CSS/JS snippets).

**Comment by nowrds** [comment]
It's really annoying to add code snippets to individual pages, posts, etc. because you always have to edit them there and have no overview or control in the plugin itself. Conditionally adding snippets within the plugin in a clear interface is a must.

**Comment by deckerweb** [comment]
Yeah, a robust visually controlled conditions feature is really missing from "Code Snippets" for a long time already. Especially since you have content snippets it is a must to setup conditions for indivual post types, posts, etc. ---> I am able to do that all by myself that is not the problem. The issue here is: it is so much faster to click it visually and avoids typing errors for such stuff especially when you are in stress...

Also for the functions conditionals make sense but for content, css and js, it should be added very soon.

Further enhancement for Pro Version:
And for single posts and pages (and post types) you could add also snippets editor in the single edit view like WPCode does. – And you could then list those "Page Snippets" in the overview table with special markup so it is clearly seen as such a thing.
